classes/class-p4-elasticsearch.php

```diff
@@ -336,6 +328,8 @@ protected function get_posts( $paged = 1 ) : array {
 								];
 								break;
 							case 'ptype':
+								// This taxonomy is used only for Posts.
+								$args['post_type']   = 'post';
```

Comment:
The whole 'filtering check' could be extracted to it's own function, to unclutter the function. What do you think @angtheod ?
https://github.com/greenpeace/planet4-master-theme/blob/f65a0dd355a66609cf1eafc637bee2167b2d679f/classes/class-p4-elasticsearch.php#L312